### PR TITLE
add Console commands for Consensus Precompiled Contract

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/consensus.go  1
- precompiled/utils.go  3
- cmd/commandline/utils.go  2
         

Complexity decreasing per file
==============================
+ precompiled/permission/permission_service_test.go  -2
         

Clones added
============
- cmd/commandline/consensus.go  3
         

Clones removed
==============
+ precompiled/chaingovernance/chain_governance_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5884580&bid=19258839)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/utils.go  3
- cmd/commandline/consensus.go  1
- cmd/commandline/utils.go  2
         

Complexity decreasing per file
==============================
+ precompiled/permission/permission_service_test.go  -2
         

Clones added
============
- cmd/commandline/consensus.go  3
         

Clones removed
==============
+ precompiled/chaingovernance/chain_governance_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5884580&bid=19258839)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/consensus.go  1
- client/go_client.go  1
         

Complexity decreasing per file
==============================
+ precompiled/permission/permission_service_test.go  -2
         

Clones added
============
- cmd/commandline/consensus.go  3
         

Clones removed
==============
+ precompiled/chaingovernance/chain_governance_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5884580&bid=19258839)

File: client/go_client.go [hunk 1]
```diff
@@ -38,17 +39,15 @@ type Client struct {
...
 	smCrypto          bool
 }
 
 const (
-	//V210 is node version v2.1.0
-	V210 = "2.1.0"
-	//V220 is node version v2.2.0
-	V220 = "2.2.0"
+	//V250 is node version v2.5.0
+	V250 int = 0x020500
```
Comment (bxq2011hust) [inline client/go_client.go:50]:
```suggestion
	V2_5_0 int = 0x02050000
```

File: client/go_client.go [hunk 2]
```diff
@@ -379,3 +382,17 @@ func (c *Client) GetTotalTransactionCount(ctx context.Context) ([]byte, error) {
 func (c *Client) GetSystemConfigByKey(ctx context.Context, configKey string) ([]byte, error) {
 	return c.apiHandler.GetSystemConfigByKey(ctx, c.groupID, configKey)
 }
+
+func getNodeVersion(strVersion string) (int, error) {
```
Comment (bxq2011hust) [inline client/go_client.go:386]:
```suggestion
func getVersionNumber(strVersion string) (int, error) {
```

File: client/go_client.go [hunk 3]
```diff
@@ -379,3 +382,17 @@ func (c *Client) GetTotalTransactionCount(ctx context.Context) ([]byte, error) {
...
+	strList := strings.Split(strVersion, ".")
```
Comment (bxq2011hust) [inline client/go_client.go:387]: strList length must be 3

File: client/go_client.go [hunk 4]
```diff
@@ -379,3 +382,17 @@ func (c *Client) GetTotalTransactionCount(ctx context.Context) ([]byte, error) {
...
+	var numVersion int
```
Comment (bxq2011hust) [inline client/go_client.go:388]:
```suggestion
	var numVersion int = 0
```

File: client/go_client.go [hunk 5]
```diff
@@ -379,3 +382,17 @@ func (c *Client) GetTotalTransactionCount(ctx context.Context) ([]byte, error) {
...
+	for i := 0; i < len(strList); i++ {
+		numVersion = numVersion << 8
+		num, err := strconv.Atoi(strList[i])
+		if err != nil {
+			return 0, fmt.Errorf("getNodeVersion failed, err: %v", err)
+		}
+		numVersion += num
```
Comment (bxq2011hust) [inline client/go_client.go:395]:
```suggestion
		versionNumber += num
		versionNumber = versionNumber << 8
```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- cmd/commandline/consensus.go  1
- client/go_client.go  1
         

Complexity decreasing per file
==============================
+ precompiled/permission/permission_service_test.go  -2
         

Clones added
============
- cmd/commandline/consensus.go  3
         

Clones removed
==============
+ precompiled/chaingovernance/chain_governance_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5884580&bid=19258839)

File: client/go_client.go [hunk 6]
```diff
@@ -38,17 +39,15 @@ type Client struct {
...
 	smCrypto          bool
 }
 
 const (
-	//V210 is node version v2.1.0
-	V210 = "2.1.0"
-	//V220 is node version v2.2.0
-	V220 = "2.2.0"
+	//V2_5_0 is node version v2.5.0
+	V2_5_0 int = 0x020500
```
Comment (bxq2011hust) [inline client/go_client.go:50]:
```suggestion
	V2_5_0 int = 0x02050000
```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- client/go_client.go  1
- cmd/commandline/consensus.go  1
         

Complexity decreasing per file
==============================
+ precompiled/permission/permission_service_test.go  -2
         

Clones added
============
- cmd/commandline/consensus.go  3
         

Clones removed
==============
+ precompiled/chaingovernance/chain_governance_test.go  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5884580&bid=19258839)

**Pull request merged**